src/main/java/mate/academy/spring/validation/PasswordsMatchValidator.java

```diff
@@ -9,6 +9,9 @@ public class PasswordsMatchValidator implements
     @Override
     public boolean isValid(UserRequestDto userRequestDto,
                            ConstraintValidatorContext constraintValidatorContext) {
+        if (userRequestDto == null) {
```

Comment:
DTO itself can't be null, we should check the password for null, because we can get NPE when trying to call `equals()` on it